android/battleBuddyAndroid/src/screens/stacks/Item/ItemDetailScreen.js

```diff
@@ -25,45 +25,117 @@ const Description = styled(Text)`
   text-align: justify;
 `;
 
-const ItemDetailScreen = ({navigation}) => {
-  const {item} = navigation.state.params;
-  // Placeholder data until data comes from backend
-  // TODO: Implement backend data for detail page
+const fillDataForCategory = (category, item) => {
```

Comment:
Will be more elegant in the future. This is just for current implementation of Firearms and Armor detail